### PR TITLE
feat: change locales via query string

### LINKED ISSUE #365: Force builder.decentraland.org language with an URL query string
E.G: builder.decentraland.org/?lan=es forces the language to spanish.

### PR DESCRIPTION
**Pull request by nicosantangelo**
Use `?locale="en|es|zh|"` 

Closes #365

### CODE REVIEW

File: src/modules/translation/sagas.ts [hunk 1]
```diff
@@ -1,6 +1,34 @@
...
+}
+
+function* handleStorageLoad() {
+  const currentLocale = yield select(getLocale)
+
+  const urlParams = new URLSearchParams(window.location.search)
+  const locale = urlParams.get('locale')
+  const locales = Object.keys(languages)
+
+  console.log({
```
Comment (belohlavek) [inline src/modules/translation/sagas.ts]: remove console log

File: src/modules/translation/sagas.ts [hunk 2]
```diff
@@ -1,6 +1,34 @@
...
+    locale,
+    locales,
+    currentLocale
+  })
+
+  if (locale && locale !== currentLocale && locales.includes(locale)) {
+    console.log('Hey')
```
Comment (belohlavek) [inline src/modules/translation/sagas.ts]: remove console log

**Comment by nicosantangelo** [comment]
Sorry for the messy PR, updated!

**Comment by belohlavek** [comment]
Not working for me + the locale is lost after entering a project?

**Comment by nicosantangelo** [comment]
I don't get why this worked when I tested, but it was a saga on dapps that was hanging. Updated!

**Pull request merged**